pkg/scheduler/apis/config/scheme/scheme_test.go

```diff
@@ -66,6 +66,9 @@ profiles:
   - name: ServiceAffinity
     args:
       affinityLabels: ["bar"]
+  - name: VolumeBinding
```

Comment:
Add one case to "empty and no plugin args". Basically to test default values.

Reply:
Also there are tests in `pkg/scheduler/framework/v1alpha1/framework_test.go` `TestNewFrameworkPluginDefaults`

Reply:
updated, thanks!